contentcuration/contentcuration/frontend/shared/data/changes.js

```diff
@@ -172,7 +172,7 @@ export class ChangeTracker extends EventEmitter {
    */
   stop() {
     this._isTracking = false;
-
+    this._timeout = setTimeout(this.dismiss.bind(this), this.expiry);
     // This sets the expiry, from 0 to an actual expiry. The clock is ticking...
     return this.renew(this.expiry);
```

Comment:
Not a big deal, it may make sense to have the timer set within the `renew` method.